src/Mammon.Tests/Actors/LAWorkspaceActorTests.cs

```diff
@@ -0,0 +1,115 @@
+﻿using Dapr.Actors.Runtime;
+using Mammon.Actors;
+using Mammon.Models.Workflows;
+
+namespace Mammon.Tests.Actors;
+
+public abstract class BaseUnitTests
+{
+    protected static CostCentreRuleEngine GetInstance()
+    {
+        var inMemorySettings = new List<KeyValuePair<string, string>> {
+            new(Consts.CostCentreRuleEngineFilePathConfigKey, "./Services/testCostCentreRules.json")
+        };
+
+        IConfiguration configuration = new ConfigurationBuilder()
+            .AddInMemoryCollection(inMemorySettings!)
+            .Build();
+
+        return new CostCentreRuleEngine(configuration);
+    }
+}
+
+
+[TestClass]
+[TestCategory("UnitTest")]
+public class LAWorkspaceActorTests : BaseUnitTests
+{
+    private readonly ActorHost _actorHost = ActorHost.CreateForTest(typeof(LAWorkspaceActor), "id");
+    private readonly ILogger<LAWorkspaceActor> _logger = Mock.Of<ILogger<LAWorkspaceActor>>();
+    private readonly Mock<ICostCentreService> _costCentreService = new();
```

Comment:
as discussed, we should use virtual methods here not to generate these artificial interfaces